lib/bundle/extend/os/linux/skipper.rb

```diff
@@ -4,7 +4,10 @@ module Bundle
   module Skipper
     class << self
       def skip?(entry, silent: false)
-        return generic_skip?(entry) unless [:cask, :mas].include?(entry.type)
+        unless [:cask, :mas].include?(entry.type) ||
+               (entry.type == :tap && entry.name == "homebrew/cask")
+          return generic_skip?(entry)
+        end
```

Comment:
```suggestion
        mac_only_types = [:cask, :mas]
        mac_only_taps = ["homebrew/cask"]

        unless mac_only_types.include?(entry.type) ||
               (entry.type == :tap && mac_only_taps.include?(entry.name)
          return generic_skip?(entry)
        end
```

Could move those vars outside the `skip?` method scope, too.